tests/fixtures/geojson_and_selects/v1.json

```diff
@@ -5,34 +5,56 @@
       {
         "type": "select_one locations",
         "name": "current_location",
-        "label": ["Where are you?"]
+        "label": [
+          "Where are you?"
+        ]
```

Comment:
🤔  I do not see the benefit of moving this on 3 lines instead of one since a fixture and chances are pretty small we need to add another some line to the list. We may want to create a `v2` instead.